recipes/rMATS/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo '#!/usr/bin/env python' | cat - RNASeq-MATS.py > tmp
+mv tmp RNASeq-MATS.py
+rm -rf testData test.bam.sh testRun.sh
+mkdir -p $PREFIX/bin
+cp -R $SRC_DIR $PREFIX/bin/$PKG_NAME
```

Comment:
The bin folder should only contain the actual scripts or binaries that form the frontend of the package. Any data or code should go into the corresponding folders defined in the [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard), e.g. ``$PREFIX/share/rmats.``

Also, I believe the gtf folder or any other data delivered together with rmats should not be part of the package, as this is also available at other places and not specific to rmats.